scalding-core/src/main/scala/com/twitter/scalding/typed/Sketched.scala

```diff
@@ -28,16 +54,16 @@ case class Sketched[K, V](pipe: TypedPipe[(K, V)],
   seed: Int)(implicit serialization: K => Array[Byte],
     ordering: Ordering[K])
   extends HasReducers {
+  import Sketched._
 
-  val reducers = Some(numReducers)
+  def serialize(k: K): Array[Byte] = serialization(k)
```

Comment:
actually I removed the hash() method entirely, we actually need to share the serialization function with the estimation code that happens below so we can pass the right key into the cms